pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1273,6 +1273,11 @@ self: super: {
   # Support recent versions of fast-logger.
   spacecookie = doJailbreak super.spacecookie;
 
+  # 2020-02-28 this was broken and is needed by glirc -- Kiwi
+  hookup = doJailbreak (super.hookup.override {
+    network = self.network_3_0_1_1;
+  });
```

The "jailbreak" suffices to fix the build. You don't need to pass the older version of `network` after jailbreaking the constraint.

```suggestion
  hookup = doJailbreak super.hookup;
```